packages/GuaCaml/GuaCaml.0.02/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+name: "guacaml"
+version: "0.02"
+synopsis: "GuaCaml : Generic Unspecific Algorithmic in OCaml"
+maintainer: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+authors: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+license: "LGPL-3.0 Linking Exception"
+homepage: "https://gitlab.com/boreal-ldd/guacaml"
+bug-reports: "https://gitlab.com/boreal-ldd/guacaml"
+depends: [ "ocaml" {>= "4.08.1"} ]
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.08.1"}
  "ocamlbuild" {build}
]
```
Are you sure you need "4.08.1" and not just "4.08"?
The package depends on ocamlbuild for building, currently fails with:
```
#=== ERROR while compiling GuaCaml.0.02 =======================================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.11.1 | pinned(https://gitlab.com/boreal-ldd/dagaml/-/wikis/releases/guacaml-0.02.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/GuaCaml.0.02
# command              /usr/bin/make
# exit-code            2
# env-file             ~/.opam/log/GuaCaml-23-865888.env
# output-file          ~/.opam/log/GuaCaml-23-865888.out
### output ###
# ocamlbuild -r -j 128 -use-ocamlfind \
# 	GuaCaml.cma \
# 	GuaCaml.cmi \
# 	GuaCaml.cmx \
# 	GuaCaml.cmxa \
# 	GuaCaml.cmxs
# make: ocamlbuild: Command not found
# make: *** [makefile:14: all] Error 127
```

Reply:
I made the suggested changes